### PR TITLE
Extend TypeDeclarationStructureTests with records

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: src/EditorFeatures/CSharpTest/Structure/TypeDeclarationStructureTests.cs [hunk 1]
```diff
@@ -211,6 +216,7 @@ public async Task TestStruct2(string typeKind)
         [Theory, Trait(Traits.Feature, Traits.Features.Outlining)]
         [InlineData("enum")]
         [InlineData("class")]
+        [InlineData("record")]
         [InlineData("struct")]
         [InlineData("interface")]
         public async Task TestStruct3(string typeKind)
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/Structure/TypeDeclarationStructureTests.cs:222]: i don't really even get how this test is working.

Comment (Youssef1313) [reply src/EditorFeatures/CSharpTest/Structure/TypeDeclarationStructureTests.cs:222]: I'm not sure too. Just extended it for records for completeness.

**Comment by Youssef1313** [comment]
@CyrusNajmabadi Let me know if we should close this test-only change or merge it. It's a minor change, but I thought it's good to add to prevent any possible regression.

**Comment by Youssef1313** [comment]
Ping @CyrusNajmabadi for review.

**Comment by CyrusNajmabadi** [comment]
Weird since it's a feature we aren't shipping.  But it's tests, so i don't mind :)

**Review by CyrusNajmabadi — APPROVED** [review]

**Pull request merged**